Add "active" access restriction (#3993)
PrairieLearn/PrairieLearn: docs/faq.md

    ...
    {
        "startDate": "2015-01-19T00:00:01",
        "mode": "Public",
        "active": false
    }
]
```

> Students who wrote the exam will then have public access to their exams after the `startDate` until the end of the course instance, while students who did not write the exam will not be able to view it. Students with access will not be able to answer questions for further credit (there is no `"credit": 100` line), but they will be able to see the entire exam in exactly the same state as when they were doing the exam originally. Because students have public access to the exam, it should be assumed that all the questions will be posted to websites such as Chegg and Course Hero. To let students see their exams with some additional security, consider only allowing (limited access post-exam under controlled conditions)[#should-students-be-able-to-review-their-exams-after-they-are-over] (although this requires in-person access by students and doesn't work online).

Reviewer: I'm afraid that there is significant regional variation in the verb used for students answering questions on a test, with "take an exam" (US, UK, Australia), "sit an exam" (UK, Australia), "write an exam" (Canada), and "give an exam" (India) all being used. It's especially confusing because the US uses both "write an exam" and "give an exam" to describe the actions taken by the _professor_ in preparing and delivering the exam.

In these situations PL defaults to US conventions, just to pick a standard.

```suggestion
Students who took the exam will then have public access to their exams after the `startDate` until the end of the course instance, while students who did not write the exam will not be able to view it. Students will not be able to answer questions (due to `"active": false`), but they will be able to see the entire exam in exactly the same state as when they were doing the exam originally. Because students have public access to the exam, it should be assumed that all the questions will be posted to websites such as Chegg and Course Hero. To let students see their exams with some additional security, consider only allowing (limited access post-exam under controlled conditions)[#should-students-be-able-to-review-their-exams-after-they-are-over] (although this requires in-person access by students and doesn't work online).
```
